app/services/to_collection_form/mapper.rb

```diff
@@ -27,8 +28,16 @@ def params
         description: CocinaSupport.abstract_for(cocina_object:), # Cocina abstract maps to Collection description
         contact_emails_attributes: CocinaSupport.contact_emails_for(cocina_object:),
         related_links_attributes: CocinaSupport.related_links_for(cocina_object:),
+        managers_attributes: participant_attributes(:managers),
+        depositors_attributes: participant_attributes(:depositors),
         version: cocina_object.version
       }
     end
+
+    def participant_attributes(role)
```

Comment:
Can you comment the code? (For me, using `.send` is magic, so always begs a comment.)